plutus-core/plutus-core/test/Spec.hs

```diff
@@ -131,7 +132,10 @@ reprint = BSL.fromStrict . encodeUtf8 . displayPlcDef
    because there are only three possibilities (@()@, @false@, and @true@). -}
 testLexConstant :: Assertion
 testLexConstant =
-    mapM_ (\t -> (fmap void . parseTerm . reprint $ t) @?= Right t) smallConsts
+    mapM_
+        (\t ->
+            (fmap
+                void . (parseTerm :: BSL.ByteString -> Either (ParseErrorBundle T.Text ParserError) (Term TyName Name DefaultUni DefaultFun SourcePos)). reprint $ t) @?= Right t) smallConsts
```

Comment:
generally I would avoid really long inline type signatures like this. If you need to do this, then pull it out to a local binding.